docs/source/examples/BSTLD.rst

```diff
@@ -5,7 +5,7 @@
 This topic describes how to manage the "BSTLD" dataset.
```

Comment:
better to avoid the `'` in the commit message
our open datasets should spell as `Open Datasets`